powertools-metrics/src/test/java/software/amazon/lambda/powertools/metrics/internal/LambdaMetricsAspectTest.java

```diff
@@ -80,7 +79,7 @@ public void metricsWithoutColdStart() {
             mocked.when(() -> SystemWrapper.getenv("AWS_EMF_ENVIRONMENT")).thenReturn("Lambda");
             internalWrapper.when(() -> getenv("_X_AMZN_TRACE_ID")).thenReturn("Root=1-5759e988-bd862e3fe1be46a994272793;Parent=53995c3f42cd8ad8;Sampled=1\"");
 
-            MetricsUtils.defaultDimensionSet(new DimensionSet());
+            MetricsUtils.defaultDimensions(null);
```

Comment:
I am not quite sure I follow what setting it to `null` means here? if no dimensions, shouldn't it just be `MetricsUtils.defaultDimensions()`? or is this because of getter/setter having the same name? If so, then the case for setting no dimensions as a default is a bit complicated UX-wise.

Reply:
Its more of a limitation of how MockedStatic works and test specific.

Ux for setting no dimension is now added as part of a test as well